aws-android-sdk-ddb-mapper/src/main/java/com/amazonaws/mobileconnectors/dynamodbv2/dynamodbmapper/DynamoDBQueryExpression.java

```diff
@@ -37,6 +37,7 @@
     private Map<String, AttributeValue> exclusiveStartKey;
     private Integer limit;
     private String indexName;
+    private String keyConditionExpression;
```

Comment:
Does this object have `equals()`, hashCode()`, `toString()` that we need to update, too?

Reply:
nothing was there originally, so will not implement them now either